apps/frontend-manage/src/components/courses/modals/CourseDeletionItem.tsx

```diff
@@ -0,0 +1,40 @@
+import { faCheck, faExclamationCircle } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { Button } from '@uzh-bf/design-system'
+import { useTranslations } from 'next-intl'
+
+function CourseDeletionItem({
+  label,
+  confirmed,
+  onClick,
+  data,
+}: {
+  label: string
+  confirmed: boolean
+  onClick: () => void
+  data?: { cy?: string; test?: string }
+}) {
+  const t = useTranslations()
+
+  return (
+    <div className="flex h-10 flex-row items-center justify-between border-b pb-2">
+      <div className="flex flex-row items-center gap-4">
+        <FontAwesomeIcon icon={faExclamationCircle} className="text-red-600" />
```

Comment:
maybe we could have the checkmark here when it was confirmed and replace this icon with it?

Reply:
Once the user confirms it, the confirm button is already replaced through a checkmark. However, I will improve the UX experience that the red highlighting vanishes at least on confirmation to properly emphasize the elements that are still to be confirmed